pkg/security/podsecuritypolicy/v1beta1/provider.go

```diff
@@ -339,19 +339,15 @@ func (s *simpleProvider) ValidateContainer(pod *api.Pod, container *api.Containe
 	}
 
 	allowEscalation := sc.AllowPrivilegeEscalation()
-	if !s.psp.Spec.AllowPrivilegeEscalation && allowEscalation == nil {
```

Comment:
let's not mix changes like this into this PR. defaulting ensures AllowPrivilegeEscalation is non-nil. if you want to pursue this anyway, let's open a separate PR to discuss it.

Reply:
> defaulting ensures AllowPrivilegeEscalation is non-nil. 

i added this to prevent nil panic. but yeah, ci still looks happy w/o this.